### ISSUE TITLE
What's up with backspace

### ISSUE THREAD

**Issue by Minothor** [BackSpace47/main#15]
Magnus, any idea what's going on? I get snowed under a bit with work and the job hunt and I come back to a news post from backspace that he's stepping down..

**Comment by BackSpace47** [comment]
I am partially back guys I am back enough that i am able to code so let me know whats up ^_^

**Comment by Chinthor** [comment]
Glad to hear it. If any of you are on ts or a server, drop me a line.

**Comment by Minothor** [comment]
I've largely stopped coding for minecraft at the moment, grew tired of the game and swapped over to starmade. looking into the possibility of writing mods for it but without a loader, it's all jarmods at the moment, a less than ideal solution.

**Comment by wolfcomp** [comment]
im on a ts server most of the time and it is private but i need to hear whit the other members/staff/stuff and hear if it is okay to hang out there (im positive we can do it :))

**Comment by BackSpace47** [comment]
I am still here if anyone has a TS server I can join I will be happy to chat with you all ^_^

**Comment by wolfcomp** [comment]
ts for those who want to come and chat PCT chann is going to be upp
tomorrow http://dragonation.ts3serv.com:10033 http://t.co/O9CINumBJ9

**Comment by wolfcomp** [comment]
has everyone died or something????

**Comment by Minothor** [comment]
Nope, I'm just employed as a forensics analyst now.

**Comment by wolfcomp** [comment]
btw we have a team page so if you work on the mod please commit to there

**Comment by Minothor** [comment]
I haven't orked on the mod in ages mate, but my code's all yours to use for
it. I may hop back some weekend, but otherwise I'm a tad snowed under in
work and projects.

**Comment by wolfcomp** [comment]
ok i will not be able to do anny thing this weekend but we hava a teamspeak and a private channel if you can hopp on next weekend i can get you permissions to the channel and stuff

**Comment by Minothor** [comment]
cheers man, I'll look into it either this weekend or next, got a Fallen
London fan game, and a few relatives to visit and help out then too.
If I get back into modding, it'll probably be either emerald casino or
symphony, if it's the latter, then I'll write in MineFacility integration
to the Harmony bridge.

**Comment by wolfcomp** [comment]
ok and im not available this week end so if there is anny one else that has server admin and is on can you say that they need to make you a channel admin of 'The PCT Channel'

**Comment by BackSpace47** [comment]
I am available at Email: backspace47@outlook.com Skype: backspace478 Facebook: Cyba Heeler

**Comment by BackSpace47** [comment]
Rofl we really need to finish this o.o Need to update it to the latest version x.x it will break everything lulz. But it will at least not be in limbo anymore o.o Probably should update the textures so they are not all over the place. Standard 16x16 should do that way the old textures can be compiled into a texture pack for higher res.

**Comment by BackSpace47** [comment]
Any chance we can all set a date and time to meet in a chat? That way we can actually see where this blob of semi-disappointment is going lol

**Comment by Minothor** [comment]
Any chance that if we do take this somewhere,  we do a hard reset now and
proceed with a cohesive plan and roles that play to our strengths?

- Nick

Skype: njboyd@gmail.com
LinkedIn: https://uk.linkedin.com/in/nickjboyd
GitHub: https://github.com/Minothor

**Comment by BackSpace47** [comment]
Yeah that actually sounds a lot better. I am not a huge fan Minecraft anymore however, coding for it is different.

**Comment by BackSpace47** [comment]
Id be game for coding from scratch. Id do graphics. I know a little Java haha, however its really messy. I can do basic things like add items and blocks. Though its been a while so i may need a little refresher course; this is fine however as it should only take a few hours >.>. I am really glad to hear that there are others of PDOXTeam still wanting to finish this mess.
